content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -833,7 +846,8 @@ private void generateSimilarityReport(String reportId, String assignmentRef) thr
 		reportData.put("view_settings", viewSettings);
 		
 		Map<String, Object> indexingSettings = new HashMap<>();
-		indexingSettings.put("add_to_index", "true".equals(assignmentSettings.get("store_inst_index")));
+		boolean addToIndex = indexingEnabled && "true".equals(assignmentSettings.get("store_inst_index"));
+		indexingSettings.put("add_to_index", addToIndex);
```

Comment:
could keep this as one line:
`indexingSettings.put("add_to_index", indexingEnabled && "true".equals(assignmentSettings.get("store_inst_index")));`

Reply:
A matter of style, I tend to find declaring a variable for a logical step more legible than performing logic within collection / map operations. I'll keep it as is unless someone chimes in preferring a one-liner